### PR TITLE
Trial button to signup, add GitHub buttons

### PR DESCRIPTION
**Pull request by SeanPackham**
screen shot 2018-03-01 at 11 11 29

### CODE REVIEW

**Comment by SeanPackham** [comment]
Louise can you copy edit my new "Join the Community" paragraph.

File: index.html [hunk 1]
```diff
@@ -34,20 +34,23 @@
   <div class="wrapper">
     <h1>Open Source Communication</h1>
     <div class="left">
-      <h2>On-Premises</h2>
+      <h2>Join the Community</h2>
       <p>
-        Run Rocket.Chat on your infrastructure<br>
-        Complete control of your data and platform
+        With more than <strong>850+ contributors</strong>, we are the largest and most active open source team chat community
```
Comment (louisemessage) [inline index.html]: With more than 850 open source contributors, we are the largest and most active team chat community

File: index.html [hunk 2]
```diff
@@ -34,20 +34,23 @@
...
-      <a class="button primary" href="/download">Install Rocket.Chat</a>
     </div>
     <div class="right">
       <h2>On Rocket.Chat Cloud</h2>
       <p>
         Run Rocket.Chat on our scalable &amp; secure cloud <br>
         Open source without the burden of infrastructure
       </p>
-      <a class="button secondary" href="/cloud#pricing">Start Your Free Trial</a>
+      <a class="button secondary" target="_blank" href="https://cloud.rocket.chat/trial/small">Start Your Free Trial</a>
```
Comment (louisemessage) [inline index.html]: Start your free trial

File: index.html [hunk 3]
```diff
@@ -63,7 +66,7 @@ <h2>Open Source</h2>
         Rocket.Chat is the leading free open source team chat Slack alternative.
       </p>
       <p>
-        With <strong><span class="star_count">{{site.stars}}k</span> stars on GitHub</strong> and more than <strong>600 contributors</strong>
+        With <strong><span class="star_count">{{ site.stars }}k</span> stars on GitHub</strong> and more than <strong>850 contributors</strong>
         across our products, we are the largest and most active open source team chat community.
```
Comment (louisemessage) [inline index.html]: ...and more than 850 open source contributors across our products, we are the largest and most active team chat community

File: index.html [hunk 4]
```diff
@@ -34,20 +34,23 @@
...
+        <a class="github-button" href="https://github.com/RocketChat/Rocket.Chat" data-icon="octicon-star" data-size="large" data-show-count="true" aria-label="Star RocketChat/Rocket.Chat on GitHub">Star</a>
+        &nbsp;
+        <a class="github-button" href="https://github.com/RocketChat/Rocket.Chat/fork" data-icon="octicon-repo-forked" data-size="large" data-show-count="true" aria-label="Fork RocketChat/Rocket.Chat on GitHub">Fork</a>
       </p>
-      <a class="button primary" href="/download">Install Rocket.Chat</a>
     </div>
     <div class="right">
       <h2>On Rocket.Chat Cloud</h2>
       <p>
         Run Rocket.Chat on our scalable &amp; secure cloud <br>
```
Comment (louisemessage) [inline index.html]: Run Rocket.Chat on our secure and scalable cloud
Open source without the burden of infrastructure

**Pull request merged**